backend/src/main/java/park_su_park/backend/request_dto/RequestUserDto.java

```diff
@@ -0,0 +1,12 @@
+package park_su_park.backend.request_dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class RequestUserDto {
+    private String username;
+    private String password;
+    private String email;
+}
```

Comment:
요청 DTO 클래스에서 validation 어노테이션을 사용하는게 좋을것 같습니다.

```java
public class RequestUserDto {
    @NotBlank
    private String username;

    @NotBlank
    private String rawPassword;

    @Email
    @NotBlank
    private String email;
}
```
뒤에 어차피 다양한 예외처리 작업할때 ( ex. 할일 제목은 10글자 이내, 유저명은 4글자 이내) 그때 같이 수정해도 될것 같습니다

Reply:
@rlatnfla 
markdown code 작성하실 때 
````
```suggestion
{내용물}
```
````
로 작성하신다면 원본과 비교하면서 보여줍니다!

Reply:
오호라